src/utilities.ts

```diff
@@ -152,93 +158,133 @@ export function getChecksumCommand() {
   return globalCheckSumCommand;
 }
 
-export function loadLaunchFile(jsonPath: string): any {
+export interface jsonDataType {
+  jsonData:any;
```

Comment:
Commented: I wanted this not to be `any`, but `jsonc.parse` _does_ return an `any` 😞